authors/apps/bookmarks/serializers.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import serializers
+from authors.apps.profiles.models import Profile
+from authors.apps.articles.serializers import ArticleSerializer
+from authors.apps.profiles.serializers import GetProfileSerializer
```

Comment:
imports from lines 2-4 are not use and so can also be omitted

Reply:
Imports have been updated.